docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,15 @@ weight: 35
 
 ## Unreleased
 
+## [v0.10.1](https://github.com/aws/eks-anywhere/releases/tag/v0.10.1)
+
+### Changed
+- Updated EKS Distro versions to latest release
```

Comment:
Do we need to update the [Artifacts ](https://anywhere.eks.amazonaws.com/docs/reference/artifacts/) in the docs to reflect new Ubuntu and Bottlerocket OS images for the release?

Reply:
Yes! will do in a follow up.